### DISCUSSION TITLE
can't mark a reply as an "accepted answer"

### DISCUSSION THREAD

**Discussion by chriscamacho** [community/community#3676] [Discussions] [answered]
in a discussion I'm sure there were a bunch of annoying pop ups saying I should mark a reply as an accepted answer, can't see any way of doing this....

**Comment by PranamBhat** [comment]
Hi @chriscamacho 

The Discussion must, first and foremost, be in a category that is question and answer : 

[https://docs.github.com/en/free-pro-team@latest/discussions/managing-discussions-for-your-community/managing-categories-for-discussions-in-your-repository#default-categories](https://docs.github.com/en/free-pro-team@latest/discussions/managing-discussions-for-your-community/managing-categories-for-discussions-in-your-repository#default-categories)

Repo owners can mark Discussions as answers as well as triagers, maintainers and the author of a post.

**Comment by chriscamacho** [comment]
they why all they annoying popups saying it should be marked answered ?

On Mon, 5 Apr 2021 at 12:54, Pranam K ***@***.***> wrote:

> Hi @chriscamacho <https://github.com/chriscamacho>
>
> The Discussion must, first and foremost, be in a category that is question
> and answer :
>
>
> ***@***.***/discussions/managing-discussions-for-your-community/managing-categories-for-discussions-in-your-repository#default-categories
>
> Repo owners can mark Discussions as answers as well as triagers,
> maintainers and the author of a post.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/github/feedback/discussions/3676#discussioncomment-570138>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFO4SP7PVWKQFZ2ZFX5J5DTHGQHHANCNFSM42I524ZQ>
> .
>


-- 

blog bedroomcoders.co.uk <http://bedroomcoders.co.uk>
Free source code and tutorials!


Disclaimer:
By sending an email to ANY of my addresses you are agreeing that:

   1. I am by definition, "the intended recipient"

   2. All information in the email is mine to do with as I see fit and make
such financial profit, political mileage, or good joke as it lends itself
to. In particular, I may quote it where I please.

   3. I may take the contents as representing the views of your company.

   4. This overrides any disclaimer or statement of confidentiality that
may be included on your message.

**Comment by jricciardi** [comment, marked as answer]
Heya @chriscamacho - you can now mark threaded replies as answers in Discussions. Check out the [changelog post for more details](https://github.blog/changelog/2023-04-17-marking-a-threaded-comment-as-the-answer/)

**Reply by offneti** [reply]
.

**Comment by usman-302** [comment]
they why all they annoying popups saying it should be marked answered ?

**Comment by usman-302** [comment]
they why all they annoying popups saying it should be marked answered ? answer me and mark
